.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy production app to Firebase App Distribution
+on:
+  push:
+    branches:
+      - main
+      - feature/6-set-up-cd-android
+
+jobs:
+  deploy_android:
+    name: Deploy Android app to Firebase App Distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    defaults:
+      run:
+        working-directory: ./sample
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Checkout source code
+        uses: actions/checkout@v2.3.2
+
+      - name: Cache Gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Decode singing properties
+        run: echo ${{ secrets.ANDROID_SIGNING_PROPERTIES }} | base64 --decode > signing.properties
+
+      - name: Restore release keystore
+        run: |
+          echo "${{ secrets.ANDROID_RELEASE_KEYSTORE }}" > release.keystore.gpg
+          gpg --batch --passphrase "${{ secrets.ANDROID_RELEASE_KEYSTORE_PASSPHRASE }}" -d -o config/release.keystore release.keystore.gpg
```

Comment:
By using secrets `ANDROID_RELEASE_KEYSTORE`, we do not need to encrypt with gpg 🙏 

Can use base64 instead https://github.com/nimblehq/flutter-templates/blob/8f165474f75bbea6249e380b2299f2ca8cccbcc8/.github/workflows/android_deploy_production.yml#L48-L50 💡

Reply:
@luongvo I just follow[ this article](https://stefma.medium.com/how-to-store-a-android-keystore-safely-on-github-actions-f0cef9413784). Do you think to encrypt with base64 is sufficient? 🤔  If so, I'll update it.

Reply:
@lydiasama I have the same concern here. So if the ANDROID_RELEASE_KEYSTORE_PASSPHRASE is compromised in the log or in the same way as the ANDROID_RELEASE_KEYSTORE as base64 only, the issue is still there 🤷‍♂️ 

I'm not 💯 confident with his approach 🙏 

![image](https://github.com/nimblehq/kmm-templates/assets/16315358/c2228c03-a13b-4501-9d98-966f59518c6d)